app/views/rooms/_menu.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="card mt-4 w-10" style="height: 10rem">
+        <ul class="list-group list-group-flush">
+            <li class="list-group-item">
+                <%= link_to "Listings", listing_room_path %>
+                <% if @room.home_type? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Price", price_room_path %>
+                <% if @room.price? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Description", description_room_path %>
+                <% if @room.listing_name? || @room.summery? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Photos", photos_room_path %>
+                <% if @room.images.any? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Amenities", amenities_room_path %>
+                <% if @room.tv == true || @room.kitchen == true %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% elsif @room.internet == true || @room.heating == true %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% elsif @room.air_conditioning == true %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Location", location_room_path %>
+                <% if @room.is_active? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item text-center">
+                <%= link_to "Publish", href="#", class: "btn btn-dark w-10" %>
```

Comment:
please update the publish button as a functional submit button

_HINT:_
You will have to create a hidden form with the `hidden_field` containing  `is_active` and the value of  `true`
then for the publish button, it should be disabled if all the checking requirements are not met

checking requirements are the following:

- room description
- room name
- room price
- room address
- room photos
- home type
- room type
- bedroom count
- bathroom count
- accomodate

once all of those are there, by then we can click the submit button